src/test/kotlin/apply/ui/api/UserRestControllerTest.kt

```diff
@@ -118,6 +123,26 @@ internal class UserRestControllerTest : RestControllerTest() {
         }.andExpect {
             status { isOk }
             content { json(objectMapper.writeValueAsString(ApiResponse.success(VALID_TOKEN))) }
+        }.andDo {
+            handle(
+                    document(
+                            "user-register",
+                            PayloadDocumentation.requestFields(
```

Comment:
```suggestion
        }.andDocument(
            "user-register",
            requestFields(
```

`fun ResultActionsDsl.andDocument(identifier: String, vararg snippets: Snippet): ResultActionsDsl {`
이런 확장함수를 만들면 반복되는 코드를 줄이고 depth도 줄일 수 있을 거 같아요 😄 

------ 

Snippet을 생성하는 DSL을 만들면
```kotlin
        }.andDocument("user-register") {
            requestFields(
```
이렇게 코틀린 DSL스러운 코드도 만들 수도 있는데 이 부분은 노가다가 좀 필요해서 시간이 남으면 반영해봐도 좋을 것 같네요.

Reply:
범블비가 제시해준 코드 바탕으로 확장함수 생성 및 기존 코드 반복 제거하였습니다!
```kotlin
fun ResultActionsDsl.andDocument(identifier: String, vararg snippets: Snippet): ResultActionsDsl {
    return andDo {
        handle(
            document(
                identifier,
                *snippets
            )
        )
    }
}
```